pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openhab</groupId>
+  <artifactId>openhab-super-pom</artifactId>
+  <version>1.0.0</version>
+
+  <packaging>pom</packaging>
+  <name>openHAB Super POM</name>
+  <description>This is a super POM for projects maintained by the openHAB Community. It contains only version-unrelated infrastructure settings and can/should also be used by projects that don't flow with the openHAB Release cycle. Settings within this POM must not have any influence on the build result.</description>
+
+  <organization>
+    <name>openHAB.org</name>
+    <url>http://www.openhab.org</url>
+  </organization>
+
+  <licenses>
+    <license>
+      <name>Eclipse Public License 2.0</name>
+      <url>http://www.eclipse.org/legal/epl-v20.html</url>
+    </license>
+  </licenses>
+
+  <properties>
+    <oh.scm.gitBaseUrl>https://github.com/openhab</oh.scm.gitBaseUrl>
+    <oh.repo.distBaseUrl>https://api.bintray.com/maven/openhab/mvn</oh.repo.distBaseUrl>
+    <oh.repo.releaseBaseUrl>https://openhab.jfrog.io/openhab</oh.repo.releaseBaseUrl>
+    <oh.repo.snapshotBaseUrl>${oh.repo.releaseBaseUrl}</oh.repo.snapshotBaseUrl>
+  </properties>
+
+  <!-- This POM must not have a <dependency> section, for the sake of reproducible builds. -->
+
+  <repositories>
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>jcenter</id>
+      <name>JCenter Repository</name>
+      <url>https://jcenter.bintray.com</url>
+    </repository>
+
+    <repository>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+      <id>openhab-artifactory-release</id>
+      <name>JFrog Artifactory Repository</name>
```

Comment:
Perhaps (WRT the comment about the ID) "openHAB Release Repository" would be a better name.